docs/syntax/quick-ref.md

```diff
@@ -0,0 +1,751 @@
+The `applies_to` tags are scope signals for readers, not comprehensive metadata. If a page contains general information that applies to all contexts, it doesn't need tags.
+:::
+
+### Page-level tag
+
+:::{dropdown} Syntax
+
+This example includes version and release phase facets, which aren't always needed. 
+
+```yaml
+---
+applies_to:
+  stack: 9.0
+  deployment:
+    ece: preview
+    eck: beta 9.0.1
+    ess: 
+    self: 9.0 
+  serverless:
+    elasticsearch:  
+    observability: deprecated 
+    security: 
+  product: 
+---
+```
+:::
+
+:::{dropdown} Output
+🚧 TODO replace this image: match example and get updated tags 🚧
+
+![annotations rendered](img/annotations.png)
+:::
+
+
+
+### Section tag
+
+:::{dropdown} Syntax
+````markdown
+# Stack-only content
+```{applies_to}
+stack: 
+```
+````
+:::
+
+:::{dropdown} Output
+#### Stack-only content
+```{applies_to}
+stack:
+```
+:::
+
+### Inline tag
+
+🚧 Coming soon 🚧
+
+**DOs**<br>
+✅ Define a set of page-level `applies_to` tags in a front matter block<br>
+✅ Add `{applies_to}` after a heading to indicate that section's contexts<br>
+✅ Indicate versions (`major.minor` with an optional `[.patch]`)  and release phases like `beta`
+
+**DON'Ts**<br>
+❌ Add `applies_to` tags to general, broadly applicable content<br>
+❌ Use `applies_to` tags as metadata or to represent "aboutness"<br>
+❌ Include `applies_to` tags in admonitions<br>
+❌ Use `Coming (x.x.x)` tags, except in special cases (don't pre-announce features)<br>
+
+
+[More details: Applies to →](applies.md)
+<br>
+<br>
+
+---
+
+## Code blocks
+
+Multi-line blocks for code, commands, configuration, and similar content. Use three backticks ` ``` ` on separate lines to start and end the block. For syntax highlighting, add a language identifier after the opening backticks.
+
+:::{dropdown} Syntax
+```markdown
+    ```yaml
+    server.host: "0.0.0.0"
+    elasticsearch.hosts: ["http://localhost:9200"]
+    ```
+```
+:::
+
+:::{dropdown} Output
+```yaml
+server.host: "0.0.0.0"
+elasticsearch.hosts: ["http://localhost:9200"]
+```
+:::
+
+
+**DOs**<br>
+✅ Include code blocks within lists or other block elements as needed<br>
+✅ Add language identifiers like `yaml`, `json`, `bash`
+
+**DON'Ts**<br>
+❌ Place code blocks in admonitions<br>
+❌ Use inline code formatting (single backticks) for multi-line content<br>
+
+[More details: Code →](code.md)
+<br>
+<br>
+
+---
+
+## Code callouts 🚧
+
+Inline annotations that highlight or explain specific lines in a code block.
+
+### Explicit callout
+To explicitly create a code callout, add a number marker in angle brackets (`<1>`, `<2>`, ...) at the end of a line. Add the corresponding callout text below the code block, in a numbered list that matches the markers.
+
+:::{dropdown} Syntax
+
+  ```
+      ```json
+      {
+        "match": {
+          "message": "search text" <1>
+        }
+      }
+      ```
+      1. Searches the `message` field for the phrase "search text"
+  ```
+  1. 🚧 rendering bug
```

Comment:
```suggestion
  ```markdown callouts=false
      ```json
      {
        "match": {
          "message": "search text" <1>
        }
      }
      ```
      1. Searches the `message` field for the phrase "search text"
  ```
  1. callout
```